python3: can't open file '/home/finley/vdepthai/depthai_demo.py': [Errno 2] No such file or directory
